node/src/test_utils/neighborhood_test_utils.rs

```diff
@@ -21,6 +22,41 @@ use std::net::Ipv4Addr;
 pub const MIN_HOPS_FOR_TEST: Hops = DEFAULT_MIN_HOPS;
 pub const DB_PATCH_SIZE_FOR_TEST: u8 = DEFAULT_MIN_HOPS as u8;
 
+lazy_static! {
+    pub static ref COUNTRY_CODE_DIGEST: Vec<(IpAddr, String, bool)> = vec![
```

Comment:
What is the point of the third value if it's always `true`? Don't you want to twiddle with it to test also other cases? (I'm not sure what it is so I'm rather expressing doubts than advising)

Edit: Eh, it's the "free world bit". Still you might want to include some countries having theirs different.